subprojects/freetype2.wrap

```diff
@@ -1,8 +1,9 @@
 [wrap-file]
-directory = freetype-2.11.1
-source_url = https://download.savannah.gnu.org/releases/freetype/freetype-2.11.1.tar.gz
-source_filename = freetype-2.11.1.tar.gz
-source_hash = f8db94d307e9c54961b39a1cc799a67d46681480696ed72ecf78d4473770f09b
+directory = freetype
+source_url = https://download.savannah.gnu.org/releases/freetype/freetype-2.12.0.tar.xz
+source_filename = freetype-2.12.0.tar.xz
+source_hash = ef5c336aacc1a079ff9262d6308d6c2a066dd4d2a905301c4adda9b354399033
 
 [provide]
 freetype2 = freetype_dep
+freetype = freetype_dep
```

Comment:
FWICT this is not correct. The pkg-config name of Freetype is `freetype2` so this should match that. Just remove this line.